src/robot/writer/filewriters.py

```diff
@@ -30,7 +30,7 @@ def FileWriter(context):
     """Creates and returns a ``FileWriter`` object.
 
     :param context: The type of the returned ``FileWriter`` is determined based
-        on ``context.format``. ``context`` is also passed to created writer.
+        on ``context.format``. ``context`` is also passed to created _writer.
```

Comment:
This doesn't look relevant. Automatic refactoring by an IDE gone wrong?

Reply:
correct